.nvmrc

```diff
@@ -0,0 +1 @@
+24
```

Comment:
umm it's been years since I stopped using vim/neovim but what's the point of this content? :-)